src/open_inwoner/openzaak/import_export.py

```diff
@@ -115,6 +117,74 @@ class CatalogusConfigImport:
     zaak_status_type_configs_imported: int = 0
     zaak_resultaat_type_configs_imported: int = 0
 
+    @staticmethod
+    def _get_url_root(url: str):
+        parsed = urlparse(url)
+        if not any((parsed.scheme, parsed.netloc)):
+            raise ValueError(f"{url} is not a valid URL")
+
+        return f"{parsed.scheme}://{parsed.netloc}"
+
+    @classmethod
+    def _lines_iter_from_jsonl_stream_or_string(cls, stream_or_string):
+        if isinstance(stream_or_string, bytes):
+            stream_or_string = stream_or_string.decode()
+        if isinstance(stream_or_string, (bytes, str)):
+            stream_or_string = stream_or_string.split("\n")
+
+        for line in stream_or_string:
+            if isinstance(line, bytes):
```

Comment:
By the time we get to this point, `stream_or_string` should be a list of strings (the result of `stream_or_string.split("\n")`), so why is this necessary?